test-cypress/integration/repository/ontop-repository.spec.js

```diff
@@ -152,6 +239,131 @@ describe('Ontop repositories', () => {
 
         // Then I expect url to be calculated properly
         OntopRepositorySteps.getUrlInput().should('have.value', 'jdbc:snowflake://someHostName.snowflakecomputing.com:1234/?warehouse=test_database');
-
     });
+
+    function assertRepositoryStatus(repositoryId, status) {
+        cy.waitUntil(() =>
+            RepositorySteps.getRepositoryFromList(repositoryId)
+                .should('be.visible')
+                .find('.repository-status .text-secondary').should('contain.text', status));
+    }
+
+    const commonRepositoryParams = {
```

Comment:
What about moving this in a fixture file?

Reply:
Fetching these parameters with `cy.fixture()` before or inside the `intercepts` caused the tests to fail. Maybe because of the file structure. In the interest of time I moved the variable to the stub file with the requests.